src/plots/Volume/avtVolumePlot.C

```diff
@@ -263,18 +268,28 @@ avtVolumePlot::SetAtts(const AttributeGroup *a)
 
     SetLegendOpacities();
 
-    double min = 0., max = 1.;
+    // SetAtts can be called before the mapper has data, so don't set
+    // legend's range unless there is input, as invalid ranges can cause
+    // vtkError messages
     if (*(mapper->GetInput()) != NULL)
-        mapper->GetRange(min, max);
-    if (atts.GetUseColorVarMin())
-    {
-        min = atts.GetColorVarMin();
-    }
-    if (atts.GetUseColorVarMax())
     {
-        max = atts.GetColorVarMax();
+        double min = 0., max = 1.;
+        mapper->GetRange(min, max);
+
+        if (atts.GetUseColorVarMin())
+        {
+            min = atts.GetColorVarMin();
+        }
+        if (atts.GetUseColorVarMax())
+        {
+            max = atts.GetColorVarMax();
+        }
+        if (min >= max)
```

Comment:
If the range is constant, min == max and this would be a valid range. Should the test be min > max?

Reply:
good catch.